src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx

```diff
@@ -0,0 +1,213 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import Access from 'app/components/acl/access';
+import AddIntegrationButton from 'app/views/organizationIntegrations/addIntegrationButton';
+import Alert from 'app/components/alert';
+import Button from 'app/components/button';
+import Confirm from 'app/components/confirm';
+import IntegrationItem from 'app/views/organizationIntegrations/integrationItem';
+import Tooltip from 'app/components/tooltip';
+import {IntegrationProvider, Integration, Organization} from 'app/types';
+import SentryTypes from 'app/sentryTypes';
+import {SingleIntegrationEvent} from 'app/utils/integrationUtil';
+
+const CONFIGURABLE_FEATURES = ['commits', 'alert-rule'];
+
+export type Props = {
+  organization: Organization;
+  provider: IntegrationProvider;
+  integration: Integration;
+  onRemove: (integration: Integration) => void;
+  onDisable: (integration: Integration) => void;
+  onReinstallIntegration: (integration: Integration) => void;
+  trackIntegrationEvent: (
+    options: Pick<SingleIntegrationEvent, 'eventKey' | 'eventName'>
+  ) => void; //analytics callback
+  className?: string;
+};
+
+//TODO: Rename to InstalledIntegration when we can remove the old one
+export default class InstalledIntegrationInDirectory extends React.Component<Props> {
```

Comment:
I copied the `InstalledIntegration` and made some small modifications to how analytics are handled. Eventually, we can delete the old component and use this one instead.